docs/proposals/ose-rebase.md

```diff
@@ -0,0 +1,39 @@
+## Rebase OSE on Origin
+
+Today, code in OSE lands by fetching and merging Origin manually. OSE-specific changes
+are scattered in the repo and it's hard to reason about any conflicts. This doc proposes
+moving all the changes in OSE that diverge from Origin into a set of commits that
+will sit on top of the Origin code[0] and a Jenkins job that will be rebasing master and
+all active release branches on a daily basis.
+
+There are two types of commits that need to be carried on top of Origin: 
+1. long-term carries related to branding and release tooling (master and release branches)
+2. CVEs (release branches only)
```

Comment:
This is not really our decision to make here, I don't think -- it's a PM/product thing. We don't want to maintain Origin releases and be expected to re-release them with more bug fixes. I guess we could ask, if we take all fixes to OSE and automatically start bringing them back, and get an automated process for build&release for both Origin and OSE, then maybe the argument is weaker against it. However, today it's a product distinction.